pkgs/development/tools/misc/debugedit/default.nix

```diff
@@ -0,0 +1,30 @@
+#TODO@deliciouslytyped The tool seems to unnecessarily force mutable access for the debugedit `-l` feature
+{fetchgit, lib, stdenv, autoreconfHook, pkg-config, elfutils, help2man, util-linux}:
+stdenv.mkDerivation {
+  name = "debugedit";
+  version = "unstable-2021-07-05";
+
+  nativeBuildInputs = [ autoreconfHook pkg-config help2man ];
+  buildInputs = [ elfutils ];
+  checkInputs = [ util-linux ]; # Tests use `rev`
+
+  src = fetchgit {
+    url = "git://sourceware.org/git/debugedit.git";
+    rev = "e04296ddf34cbc43303d7af32ab3a73ac20af51a";
+    sha256 = "19cjkpzhdn2z6fl7xw8556m6kyrb7nxwbz2rmiv2rynyp74yg44z";
+  };
+
+  preBuild = ''
+    patchShebangs scripts/find-debuginfo.in
+  '';
+
+  doCheck = true;
+
+  meta = with lib; {
+    description = "debugedit provides programs and scripts for creating debuginfo and source file distributions, collect build-ids and rewrite source paths in DWARF data for debugging, tracing and profiling.";
+    homepage = "https://sourceware.org/debugedit/";
+    license = licenses.gpl2Plus;
```

Comment:
Homepage says
> The debuginfo project as a whole can be redistributed under the GPLv3+. Some individual source files may also be redistributed under the GPLv2+ or under the LGPLv2+. See the headers of the individual files to learn which licenses apply. All licenses used are upward compatible with GPLv3+.

Reply:
Aha, I only looked at a source header. I guess GPL3+ then?

Reply:
Unless you want to check *all* source files, I suppose it's better to trust the author :)

Reply:
Yes, but I was insufficiently observant to notice that on the home page. :)